helm_deploy/prometheus-custom-rules-resettlement-passport.yaml

```diff
@@ -0,0 +1,18 @@
+apiVersion: monitoring.coreos.com/v1
+kind: PrometheusRule
+metadata:
+  namespace: hmpps-resettlement-passport-dev
+  labels:
+    role: alert-rules
+  name: hmpps-resettlement-passport-external-api-rules
```

Comment:
`hmpps-resettlement-passport-api-custom` might match the convention a bit better

Reply:
Done